pkg/services/accesscontrol/dualwrite/collectors.go

```diff
@@ -394,6 +396,35 @@ func rolePermissionsCollector(store db.DB) legacyTupleCollector {
 	}
 }
 
+// basicRoleBindingsCollector collects role bindings for basic roles
+func anonymousRoleBindingsCollector(cfg *setting.Cfg, orgService org.Service) legacyTupleCollector {
+	return func(ctx context.Context, orgID int64) (map[string]map[string]*openfgav1.TupleKey, error) {
+		tuples := make(map[string]map[string]*openfgav1.TupleKey)
+		object := zanzana.NewTupleEntry(zanzana.TypeRole, zanzana.TranslateBasicRole(cfg.AnonymousOrgRole), "")
+		// Object should be set to delete obsolete permissions
+		tuples[object] = make(map[string]*openfgav1.TupleKey)
+
+		o, err := orgService.GetByName(ctx, &org.GetOrgByNameQuery{Name: cfg.AnonymousOrgName})
+		if err != nil {
+			return tuples, nil
+		}
+
+		if o.ID != orgID {
+			return tuples, nil
+		}
+
+		tuple := &openfgav1.TupleKey{
+			User:     zanzana.NewTupleEntry(zanzana.TypeAnonymous, "0", ""),
+			Relation: zanzana.RelationAssignee,
+			Object:   object,
+		}
+
+		tuples[tuple.Object][tuple.String()] = tuple
+
+		return tuples, nil
+	}
+}
```

Comment:
Instead of using org service we should create our own query and use db, like we do with everything else here